convene-web/app/javascript/src/custom_components/form.scss

```diff
@@ -75,4 +75,17 @@
 }
 .access-code-form__submit {
   @apply mt-4;
+}
+
+.identification-form {
+  @apply w-full;
+  @screen sm {
+    @apply w-1/2;
+  }
+}
+.identification-form__submit {
```

Comment:
Not Blocking: I really need to write out what I consider "Semantic, Composable BEM" which is distinct from regular BEM in the following ways:

1. When we _can_ rely on targeting native HTML elements, such as `input[type="submit"]`, we do.
2. When we are targeting sub-components, we use CSS cascading, not `__`. This allows us to have some guarantee that we can override application level styling at the  component or sub-component level.

In this particular case, what this would mean is we would refactor to:

```scss
.identification-form { 
  input[type="submit"] {
  
  }
}
```